nucypher/blockchain/eth/agents.py

```diff
@@ -228,12 +230,34 @@ def partition_stakers_by_activity(self) -> Tuple[List[str], List[str], List[str]
 
         return active_stakers, pending_stakers, missing_stakers
 
-    def get_all_locked_tokens(self, periods: int) -> int:
-        """Returns the current period"""
+    def get_all_active_stakers(self, periods: int, pagination_size: int = None) -> Tuple[int, list]:
+        """Only stakers which confirmed the current period (in the previous period) are used."""
         if not periods > 0:
             raise ValueError("Period must be > 0")
 
-        all_locked_tokens, _stakers = self.contract.functions.getAllActiveStakers(periods).call()
+        if pagination_size is None:
+            pagination_size = StakingEscrowAgent.DEFAULT_PAGINATION_SIZE if self.blockchain.is_light else 0
```

Comment:
This is the *light mode" entry point... yeah?

Reply:
yep, I see this as - all calls that will be broken in light mode will use something like this